conan/api/model/list.py

```diff
@@ -43,11 +44,17 @@ def keep_outer(self, other):
     @staticmethod
     def load(file):
         try:
-            content = json.loads(load(file))
+            content = json.loads(load(make_abs_path(file)))
```

Comment:
Please don't change the logic here just for the sake of adding some extra error messages.